### PR TITLE
add cluster page

### PR DESCRIPTION
**Pull request by insujang**
#143

### CODE REVIEW

File: source/cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+![server](server.jpg)
+
+SymbioticLab operates powerful in-house clusters dedicated for SymbioticLab's research.
+We have two clusters specialized for memory and GPUs, respectively:
+
+### 6 Memory nodes
+- 2 x Intel Xeon Processors
+- 512GB DRAM + 512GB Intel Optane DIMM
```
Comment (jaywonchung) [inline source/cluster/index.md:13]:
```suggestion
- 512 GB DRAM + 512 GB Intel Optane DIMM
```

File: source/cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- 1 x NVIDIA A40 GPU
+
+### 6 GPU nodes
+- 2 x AMD EPYC Processors
+- 512GB DRAM
```
Comment (jaywonchung) [inline source/cluster/index.md:18]:
```suggestion
- 512 GB DRAM
```

File: source/cluster/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- 4 x NVIDIA A40 GPUs
+
+**In total, 30 NVIDIA A40 GPUs, 1440 CPU cores, and 9TB memory, all connected to each other using a 200Gbps Infiniband adaptor. The computing power is more powerful than using eight [AWS p3.8xlarge](https://aws.amazon.com/ec2/instance-types/p3/) instances, which costs $70,500 per month!**
```
Comment (jaywonchung) [inline source/cluster/index.md:21]:
```suggestion
**In total, 30 NVIDIA A40 GPUs, 1440 CPU cores, and 9 TB memory, all connected to each other using a 200 Gbps Infiniband adaptor. The computing power is more powerful than using eight [AWS p3.8xlarge](https://aws.amazon.com/ec2/instance-types/p3/) instances, which costs $70,500 per month!**
```

**Review by jaywonchung — CHANGES_REQUESTED** [review]
I did some u-nitpicking ;)

**Review by jaywonchung — COMMENTED** [review]
No more nitpicks. It just needs to be connected with some page now. Thanks for creating this Insu! 🚀

**Review by mosharaf — APPROVED** [review]
LGTM. Thanks @insujang @jaywonchung.

**Pull request merged**